srcpkgs/font-adobe-source-code-pro/template

```diff
@@ -0,0 +1,28 @@
+# Template file for 'font-adobe-source-code-pro'
+pkgname=font-adobe-source-code-pro
+version=1.010
+revision=2
```

Comment:
revision should be 1

Reply:
Updated. By the way, when is the revision supposed to be changed, anyways? I read the description in `Manual.md`:

> `revision`: A number that must be set to 1 when the `source package` is created, or updated to a new `upstream version`. This should only be increased when the generated `binary packages` have been modified.

But the description is not clear to me. Does the package stay at `revision=1` even after an update comes and the package is updated?